LoggerConfig/LoggerImplementation/AutoLog.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Discord;
+using Serilog.Events;
+using Serilog;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace QuickEdit.LoggerConfig.LoggerImplementation;
+
+public class AutoLog
+{
+    // skipcq: CS-R1073
```

Comment:
Explain why the error is skipped

```suggestion
    // LogMessage does not run asynchronously, so we can ignore the DeepSource error
    // skipcq: CS-R1073
```